### PR TITLE
[17/N] Fix extra warnings brought by clang-tidy-17

### PR DESCRIPTION
**Pull request by cyyever**
Fixes #ISSUE_NUMBER

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10

### CODE REVIEW

**Review by Skylion007 — APPROVED** [review]

File: aten/src/ATen/native/mkldnn/xpu/Conv.cpp [hunk 1]
```diff
@@ -16,19 +16,18 @@ using namespace dnnl;
...
   std::vector<int64_t> stride;
   std::vector<int64_t> padding;
   std::vector<int64_t> dilation;
-  bool transposed;
+  bool transposed{};
   std::vector<int64_t> output_padding;
-  int groups;
-  bool benchmark;
-  bool deterministic;
+  int64_t groups{};
```
Comment (Skylion007) [inline aten/src/ATen/native/mkldnn/xpu/Conv.cpp:28]: We really support int64 groups?

Comment (cyyever) [reply aten/src/ATen/native/mkldnn/xpu/Conv.cpp:28]: Should revert, changed by clang-tidy

**Pull request closed without merging**